modules/plugins/theme/theme.nix

```diff
@@ -13,13 +13,16 @@
   supportedThemes = import ./supported-themes.nix {
     inherit lib config;
   };
+  base16-colors = import ./base16-colors.nix {
+    inherit lib;
+  };
```

Comment:
I don't know how I feel about importing this, you could just make a helper function and write them here